lib/perl/Genome/VariantReporting/Framework/Component/Report/MergedReport.pm

```diff
@@ -277,16 +276,14 @@ sub can_be_merged {
 
 sub merge_parameters {
     my $self = shift;
-    my @report_results = $self->report_results;
-    my $result_class = $report_results[0]->class;
+    my $result_class = $self->base_report->class;
     return $result_class->merge_parameters;
```

Comment:
I probably wrote this originally, but I don't remember - is there a reason why we have to call merge_parameters on the class and don't just cal $self->base_report->merge_parameters?